sycl/include/CL/sycl/detail/pi.def

```diff
@@ -18,25 +18,30 @@
 _PI_API(piPlatformsGet)
 _PI_API(piPlatformGetInfo)
 // Device
-_PI_API(piextDeviceConvert)
 _PI_API(piDevicesGet)
 _PI_API(piDeviceGetInfo)
 _PI_API(piDevicePartition)
 _PI_API(piDeviceRetain)
 _PI_API(piDeviceRelease)
 _PI_API(piextDeviceSelectBinary)
 _PI_API(piextGetDeviceFunctionPointer)
+_PI_API(piextDeviceGetNative)
+_PI_API(piextDeviceFromNative)
```

Comment:
The "FromNative" does not have an action verb. How about piext*GetNativeHandle & piext*CreateWithNativeHandle"?

Reply:
Good point! I was contemplating the naming but was afraid of getting too verbose, but given that we have similar naming for some of the program creation functions, I think your suggestion fits nicely.